.github/workflows/ci.yml

```
@@ -313,6 +313,22 @@ jobs:
       RUST_BACKTRACE: 1
       TRYBUILD: overwrite
 
+  docsrs:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'CI-build-full') || (github.event_name != 'pull_request' && github.ref != 'refs/heads/main') }}
+    needs: [fmt]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v4
+      - uses: Swatinem/rust-cache@v2
+        with:
+          key: cargo-careful
+        continue-on-error: true
+      - uses: dtolnay/rust-toolchain@nightly
+        with:
+          components: rust-src
+      - run: cargo rustdoc --lib --no-default-features --features "macros num-bigint num-complex hashbrown serde multiple-pymethods indexmap eyre either chrono rust_decimal" -Zunstable-options --config "build.rustdocflags=[\"--cfg\", \"docsrs\"]"
```

It looks like the difference between this and `full` is that it doesn't include `experimental-inspect` but does include `multiple-pymethods`.

To help keep things consistent should we take `experimental-inspect` out of `full` (and manually add it back in to relevant CI jobs) and then make this 

```suggestion
      - run: cargo rustdoc --lib --no-default-features --features full -Zunstable-options --config "build.rustdocflags=[\"--cfg\", \"docsrs\"]"
```

(Or `--features "full multiple-pymethods"`, but I think `multiple-pymethods` has no documentation differences, just internal machinery.)

---

(Or maybe we should be documenting `experimental-inspect` also; it might help people notice the feature and motivate someone to come along and help finish it off.)

---

I think documenting `experimental-inspect` would be preferable. Or rather, I think the only reason to not document a feature is if it breaks the docs.rs build or we do not want people to use it, neither of which appears to be the case here.

---

Added a commit unifying the docs.rs and full builds.